src/commcare_cloud/terraform/modules/network/main.tf

```diff
@@ -331,6 +331,13 @@ resource "aws_security_group" "db-private" {
     ]
   }
 
+  ingress {
+    from_port   = 6432
+    to_port     = 6432
+    protocol    = "tcp"
+    cidr_blocks = aws_subnet.subnet-db-private.*.cidr_block
```

Comment:
I think this is terraform 0.12 syntax that's not supported on terraform 0.11, so it's a situation where you'll have to make a PR for 0.11 and then fix it in the `terraform-v12` branch once master's merged into that. Or something along those lines

Reply:
Found the fix for this Danny. I will raise a new PR.

Reply:
PR: https://github.com/dimagi/commcare-cloud/pull/4760